exchanges/exchanges.go

```diff
@@ -1616,38 +1639,37 @@ func (bittrex *BittrexExchange) msgHandler(msg signalr.Message) {
 // bittrex order book, they will periodically send empty updates, which will
 // trigger the full order book request.
 func (bittrex *BittrexExchange) connectWs() {
-	err := bittrex.connectSignalr(&signalrConfig{
-		host:           BittrexURLs.Websocket,
-		protocol:       "1.5",
-		endpoint:       "/signalr",
-		connectionData: `[{"name":"c2"}]`,
-		params:         nil,
-		msgHandler:     bittrex.msgHandler,
-	})
 	bittrex.orderMtx.Lock()
 	bittrex.queue = make([]*BittrexOrderbookUpdate, 0)
 	bittrex.orderSeq = 0
 	bittrex.orderMtx.Unlock()
+
+	err := bittrex.connectSignalr(&signalrConfig{
+		host:       "socket-v3.bittrex.com",
+		protocol:   "1.5",
+		endpoint:   "/signalr",
+		msgHandler: bittrex.msgHandler,
+	})
```

Comment:
Thoughts about migrating the signalr repo to https://github.com/rainhq/signalr from the archived carterjones one? I was poking around and it seems active and even updated with [an example for bittrex v3](https://github.com/rainhq/signalr/blob/v2/examples/bittrex_v3/main.go).

Reply:
Looks like that should work. Do you mind if I open an issue and follow up with the change? I wanna hit some other priorities first.

Reply:
Yah, sounds good